plugins/powerd/fu-plugin-powerd.c

```diff
@@ -73,6 +73,10 @@ fu_plugin_update_prepare (FuPlugin *plugin,
 	gdouble current_level = 0;
 	g_autoptr(GVariant) powerd_response = NULL;
 
+	/* permit updates when the device does not care about power conditions */
```

Comment:
rebase on top of previous change so you can remove these lines from the current location on this change

Reply:
Thanks!